### PR TITLE
doc: fix pdclust link at CORTX-MOTR-ARCHITECTURE.md

### PR DESCRIPTION
**Pull request by johnbent**
Fix the link to parity declustered design document.

-----
[View rendered doc/CORTX-MOTR-ARCHITECTURE.md](https://github.com/Seagate/cortx-motr/blob/johnbent-patch-4/doc/CORTX-MOTR-ARCHITECTURE.md)

### CODE REVIEW

File: doc/CORTX-MOTR-ARCHITECTURE.md [hunk 1]
```diff
@@ -75,6 +75,7 @@
 # Object Layout
 + Object is an array of blocks. Arbitrary scatter-gather IO with overwrite. Object has layout.
 + Default layout is parity de-clustered network raid: N+K+S striping.
+   + More details about [parity declustering](https://github.com/Seagate/cortx-motr/blob/documentation/doc/pdclust/index.rst)
```
[the remark this answers is not in the document]
Comment (nikitadanilov) [reply doc/CORTX-MOTR-ARCHITECTURE.md:78]: Agree, let's move this to main and them re-issue this PR.

**Review by nikitadanilov — CHANGES_REQUESTED** [review]

File: doc/CORTX-MOTR-ARCHITECTURE.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (nikitadanilov) [reply doc/CORTX-MOTR-ARCHITECTURE.md:78]: https://github.com/Seagate/cortx-motr/blob/documentation/doc/sm/index.rst can also be moved to main: it's another deep dive document.

**Review by nikitadanilov — APPROVED** [review]

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

**Review by andriytk — APPROVED** [review]

**Comment by andriytk** [comment]
@mehjoshi, it can be landed now, i think.

**Comment by mehjoshi** [comment]
> @mehjoshi, it can be landed now, i think.

Sure.
@rkothiya, Could you please merge this PR once the deployment and other checks complete? Thanks.

**Comment by rkothiya** [comment]
giving an exception for pre-merge and ci/cd regression job as this is related to documentation.

**Pull request merged**